gateway/cli/src/main.rs

```diff
@@ -236,11 +236,21 @@ pub async fn print_response(response: reqwest::Response) {
 }
 
 pub fn source_password(rpcpassword: Option<String>) -> String {
-    match rpcpassword {
-        None => rpassword::prompt_password("Enter gateway password:").unwrap(),
```

Comment:
If this `unwrap` was the problem we'd see a stack trace in #1437, the actual password checking happens elsewhere. As the function name implies, this function merely returns the password and doesn't check it, so this PR doesn't fix the problem.